### ISSUE TITLE
vue-cli-service-build-target-lib-loses-images-path-when-imported-as-lib

### ISSUE THREAD

**Issue by johnsky123** [vuejs/vue-cli#4100]
### Version
3.5.0

### Reproduction link
[https://stackoverflow.com/questions/51523450/vue-cli-service-build-target-lib-loses-images-path-when-imported-as-lib](https://stackoverflow.com/questions/51523450/vue-cli-service-build-target-lib-loses-images-path-when-imported-as-lib)

### Environment info
```
mac os
```

### Steps to reproduce
There is no solution to this problem

### What is expected?
solve this problem

### What is actually happening?
I followed this guide to create my npm module. My repo contains in the src/assets folder an svg image (hue-cirle.svg).

When building with vue-cli-service build --target lib --name <lib-name> the svg image is bundled under dist/img/hue-circle123456.svg.

Now when installing the module with npm install --save <module> the path remains src/img/hue-circle123456.svg, while it should refer to the svg image from the node_modules dist folder and not to a non-existing hue-circle123456.svg image from the main Vue App src folder.

Basically what happens is that the svg image path is wrong in the Vue project that uses my module. I tried to add the ~ to force webpack to interpret it as a module dependency as suggested here but it did not work.

In the end I gave up and embedded the svg in the html template as you can see in the repo. Nonetheless, I ask this question since I could not find any solution and I, as well as others, may need to solve this in order to use image resources in custom modules built with vue-cli-service build --target lib --name <lib-name> command.

Thanks for the help!
